.github/workflows/ci.yml

```diff
@@ -129,6 +129,20 @@ jobs:
           build_args: GITHUB_TOKEN=${{ secrets.DSP_REPO_TOKEN }}
           tag_with_ref: true
 
+  publish-merlin-transformer-docker:
```

Comment:
Can you add benchmark on the component that might take time? excluding external call though